stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -3181,8 +3181,24 @@ func.func @dynamic_reshape(%arg0: tensor<?xf32>, %shape: tensor<2xindex>) -> ten
 
 // -----
 
+func.func @dynamic_reshape_c1(%arg0: tensor<?xf32>, %shape: tensor<2xindex>) -> tensor<?x?xf64> {
+  // expected-error @+1 {{expects operand and result to have compatible element type}}
+  %0 = "stablehlo.dynamic_reshape"(%arg0, %shape) : (tensor<?xf32>, tensor<2xindex>) -> tensor<?x?xf64>
+  func.return %0 : tensor<?x?xf64>
+}
+
+// -----
+
+func.func @dynamic_reshape_c2(%arg0: tensor<11xf32>, %shape: tensor<2xindex>) -> tensor<2x5xf32> {
+  // expected-error @+1 {{number of output elements (10) doesn't match expected number of elements}}
+  %0 = "stablehlo.dynamic_reshape"(%arg0, %shape) : (tensor<11xf32>, tensor<2xindex>) -> tensor<2x5xf32>
+  func.return %0 : tensor<2x5xf32>
+}
+
+// -----
```

Comment:
Can you update the name of the test that has shared verifier logic to something like:
`@dynamic_reshape_c3_reshape_c#`

Reply:
those are at ops_stablehlo_quantized.mlir. updated. Thanks.